src/openrct2-ui/windows/Window.h

```diff
@@ -80,6 +80,7 @@ rct_window* window_finances_open();
 rct_window* window_finances_research_open();
 
 // WC_PARK_INFORMATION
+rct_window* window_park_open();
```

Comment:
Why is there two of these L85 as well? This isn't even called so is pointless.

Reply:
Removed